packages/lesswrong/components/posts/PostsList2.tsx

```diff
@@ -177,6 +180,11 @@ const PostsList2 = ({
 
       <div className={boxShadow ? classes.posts : null}>
         {orderedResults && orderedResults.map((post, i) => {
+          if (isPostRepeated(post._id)) {
+            return null;
+          }
+          addPost(post._id);
```

Comment:
Dumb Q: How come this doesn't fail if we're not a descendant of a `HideRepeatedPostsProvider`?

Reply:
If we're outside of a provider then `useContext(HideRepeatedPostsContext)` will just return the default static value that we passed into `createContext` at line 8 of HideRepeatedPostsContext.tsx.